src/modify.ts

```diff
@@ -0,0 +1,34 @@
+/* eslint-disable no-restricted-syntax */
+/* eslint-disable @typescript-eslint/no-unsafe-member-access */
+import 'source-map-support/register';
+import { DynamoDBStreamEvent, Context, Callback } from 'aws-lambda';
+import logger from './observability/logger';
+import { eventHandler } from './eventHandler';
+
+const {
+  NODE_ENV, SERVICE, AWS_REGION, AWS_STAGE,
+} = process.env;
+
+logger.debug(
+  `\nRunning Service:\n '${SERVICE}'\n mode: ${NODE_ENV}\n stage: '${AWS_STAGE}'\n region: '${AWS_REGION}'\n\n`,
+);
+
+const handler = async (event: DynamoDBStreamEvent, _context: Context, callback: Callback) => {
+  try {
+    logger.debug(`Function triggered with '${JSON.stringify(event)}'.`);
+    if (!process.env.NO_MODIFY) {
```

Comment:
As discussed - could we flip this logic so that releasing this code:
- won't send update messages by default
- doesn't rely on the env/terraform change to be made to prevent the messages being sent


```suggestion
    if (process.env.SEND_BILLING_AMENDMENTS) {
```

Reply:
done